src/components/Button/Button.js

```diff
@@ -13,8 +21,9 @@ class Button extends Component {
   };
 
   render() {
+    let classes = classNames('sd-button', 'ui', 'button', this.props.color, this.props);
```

Comment:
Let's not pass in all props as classes.  We should map the support we want from the outside.